src/lib/organisms/footer.svelte

```diff
@@ -75,10 +75,7 @@
 			</ul>
 		</div>
 	</div>
-
-	<div class="left-wrapper">
-	
-	</div>
+	<div class="left-wrapper"></div>
```

Comment:
Te veel ```<div>```'s

Reply:
Footer gaan we nog refactoren dit is nog oude code. Dit gaan we zeker nog toepassen.